vuepress/docs/next/docs/curate/bundle-version-types.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Bundle Classification *
+
+Entando 7.1 implements a more modular approach to building apps with the **ent bundle** CLI orchestrating the management of project bundles. The Entando bundle was restructured, its processes decoupled by component type, and the publishing transport method was streamlined. With these changes, Entando Bundles went from git-based to Docker-based bundles, and this page clarifies the differences between the two, along with the improvements that were made in the new release.
```

Comment:
Will mention OCI type to Docker-based bundle >> Entando Bundles went from git-based to Docker-based OCI bundles